docs/file-logger.md

```diff
@@ -84,6 +87,7 @@ object FileApp extends ZIOAppDefault {
        |logger {
        |  format = "%timestamp{yyyy-MM-dd'T'HH:mm:ssZ} %fixed{7}{%level} [%fiberId] %name:%line %message %cause"
        |  path = "file:///tmp/file_app.log"
+       |  rollingPolicy = TimeBasedRollingPolicy
```

Comment:
you should modify also https://github.com/zio/zio-logging/blob/master/examples/core/src/main/scala/zio/logging/example/FileApp.scala, to match with this documentation

Reply:
I applied